File: src/sdk-reference/csharp/1/index/get-auto-refresh/snippets/getAutoRefresh.cs
try {
  bool autorefresh_flag = kuzzle.index.getAutoRefresh("nyc-open-data");

  if (autorefresh_flag) {
    Console.WriteLine("Autorefresh is enabled");
  } else {
    Console.Error.WriteLine("Autorefresh is disabled");
  }
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}

File: src/sdk/cpp/1/controllers/index/get-auto-refresh/snippets/getAutoRefresh.cpp
try {
  bool autorefresh_flag = kuzzle->index->getAutoRefresh("nyc-open-data");

  if (autorefresh_flag) {
    std::cout << "Autorefresh is enabled" << std::endl;
  } else {
    std::cerr << "Autorefresh is disabled" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
